client/app/iplb/configuration/iplb-configuration.html

```diff
@@ -1,30 +1,65 @@
-        <column title="'iplb_configuration_datacenter' | translate" property="name" sortable="asc"></column>
-        <column title="'iplb_configuration_progress' | translate"
-            sortable
-            class="oui-table__cell_component">
-            <oui-progress data-compact data-ng-show="$row.task">
-                <oui-progress-bar data-type="{{ ctrl.ServiceHelper.constructor.getTaskProgressType($row.task.status) }}" data-value="$row.task.progress">
-                </oui-progress-bar>
+    <oui-datagrid
+        data-ng-if="!$ctrl.zones.loading"
+        data-rows="$ctrl.zones.data"
+        data-selectable-rows>
+        <extra-top>
+            <oui-button
+                class="mb-3"
+                data-variant="secondary"
+                data-text="{{:: 'iplb_configuration_action_apply' | translate }}"
+                data-on-click="$ctrl.applyChanges($selectedRows)"
+                data-disabled="!$selectedRows.length"></oui-button>
+        </extra-top>
+        <oui-column
+            data-title=":: 'iplb_configuration_datacenter' | translate"
+            data-property="name"
+            data-sortable="asc"></oui-column>
+        <oui-column
+            data-title=":: 'iplb_configuration_progress' | translate"
+            data-sortable>
+            <span
+                data-ng-if="$row.task.progress === 100 && $ctrl.applications[$row.id]"
+                data-translate="iplb_configuration_changesApplied"></span>
+            <oui-progress
+                data-compact
+                data-ng-if="$row.task && $row.task.progress !== 100">
+                <oui-progress-bar
+                    data-type="{{ $ctrl.ServiceHelper.constructor.getTaskProgressType($row.task.status) }}"
+                    data-value="$row.task.progress"></oui-progress-bar>
             </oui-progress>
-        </column>
-        <column title="'iplb_configuration_status' | translate" template="ctrl.constructor.statusTemplate()" class="oui-table__cell_action"></column>
-        <column title="'iplb_configuration_action' | translate"
-            class="oui-table__cell_action oui-table__cell_action-menu"
-            template="ctrl.constructor.actionTemplate()">
-        </column>
-    </oui-table>
+        </oui-column>
+        <oui-column data-title=":: 'iplb_configuration_status' | translate">
+            <span
+                data-ng-if="$row.changes === 0"
+                class="oui-status oui-status_success"
+                data-translate="iplb_configuration_changes_0"></span>
+            <span
+                data-ng-if="$row.changes === 1"
+                class="oui-status oui-status_warning"
+                data-translate="iplb_configuration_changes_1"></span>
+            <span
+                data-ng-if="$row.changes > 1"
+                class="oui-status oui-status_warning"
+                data-translate="iplb_configuration_changes_count"
+                data-translate-values="{ count: $row.changes }"></span>
```

Comment:
these three span cannot be done with only one and ngPluralize ?

Reply:
Did try this and wasn't happy with the result !

Reply:
Done, it works great !